docs/platforms/python/integrations/celery/index.mdx

```diff
@@ -17,36 +17,89 @@ pip install --upgrade 'sentry-sdk[celery]'
 
 If you have the `celery` package in your dependencies, the Celery integration will be enabled automatically when you initialize the Sentry SDK.
 
-Make sure that the **call to `init` is loaded on worker startup**, and not only in the module where your tasks are defined. Otherwise, the initialization happens too late and events might end up not being reported.
+Make sure that the **call to `sentry_sdk.init()` is loaded on worker startup**, and not only in the module where your tasks are defined. Otherwise, the initialization happens too late and events might end up not being reported.
 
-<PlatformContent includePath="getting-started-config" />
+### Setup Celery (Without Django)
 
-### Standalone Setup
+To get the most out of Sentry make sure to initialize the Sentry SDK in your Celery worker processes as well as your application that is sending messages to Celery.
 
-If you're using Celery standalone, there are two ways to set this up:
+In addition to capturing errors, you can monitor interactions between multiple services or applications by [enabling tracing](/concepts/key-terms/tracing/). You can also collect and analyze performance profiles from real users with [profiling](/product/explore/profiling/).
```

Comment:
I agree that it's a bit jarring, but I think it's also a confusing experience to have the options to add tracing/profiling without any explanation. (Especially because we now know that a lot of people don't know what tracing is.) Maybe we could just shorten it and say: "When using Celery without Django, you'll need to initialize the Sentry SDK in both your application and the Celery worker process. 

In addition to capturing errors, you can use Sentry for [distributed tracing](/concepts/key-terms/tracing/) and [profiling](product/explore/profiling/). Select what you'd like to install to get the corresponding installation and configuration instructions below."

Reply:
Updated